pypit/arsciexp.py

```diff
@@ -16,6 +16,7 @@
 from pypit import armasters
 from pypit import armsgs
 from pypit import arproc
+from pypit import arqa
```

Comment:
I have this feeling I removed this line from up here previously.
But am not 100% sure.